packages/perseus/src/widgets/radio/prompt-utils.ts

```diff
@@ -0,0 +1,27 @@
+import {WidgetType, type RadioPromptJSON} from "../../prompt-types";
+
+import type {RenderProps} from "./radio-component";
+import type {PerseusRadioUserInput} from "../../validation.types";
+
+export const getPromptJSON = (
+    renderProps: RenderProps,
+    userInput: PerseusRadioUserInput,
+): RadioPromptJSON => {
+    const {choices} = renderProps;
+
+    const options = choices.map((choice) => {
+        return {
+            value: choice.content,
+        };
+    });
+
+    return {
+        type: WidgetType.RADIO,
```

Comment:
You can just use "radio" here. We use lower-kebab-case for widget types:

``` TS
// radio.ts
name: "radio",
```

``` TS
// numeric-input.tsx
name: "numeric-input",
```

``` suggestion
        type: "radio",
```

Reply:
I updated the enum to be lower case